app/src/main/res/menu/menu_talk.xml

```diff
@@ -17,40 +17,45 @@
         app:showAsAction="always">
         <menu>
 
-            <item
-                android:id="@+id/menu_watch"
-                android:icon="@drawable/ic_star_24"
-                android:title="@string/menu_page_watch"
-                android:visible="false"
-                app:showAsAction="never" />
             <item
                 android:id="@+id/menu_change_language"
-                android:icon="@drawable/ic_translate_toolbar_icon_color_themed"
+                android:icon="@drawable/ic_translate_white_24dp"
                 android:title="@string/langlinks_activity_title"
                 android:visible="false"
+                app:iconTint="?attr/material_theme_secondary_color"
                 app:showAsAction="never" />
             <item
                 android:id="@+id/menu_view_user_page"
-                android:icon="@drawable/ic_user_avatar_material_secondary_themed"
+                android:icon="@drawable/ic_user_avatar"
                 android:title="@string/menu_option_user_page"
                 android:visible="false"
+                app:iconTint="?attr/material_theme_secondary_color"
+                app:showAsAction="never" />
+            <item
+                android:id="@+id/menu_view_edit_history"
+                android:icon="@drawable/ic_icon_revision_history_apps"
+                android:title="@string/menu_page_edit_history"
+                app:iconTint="?attr/material_theme_secondary_color"
                 app:showAsAction="never" />
             <item
                 android:id="@+id/menu_read_article"
                 android:icon="@drawable/ic_article_ltr_ooui"
                 android:title="@string/button_continue_to_article"
                 android:visible="false"
+                app:iconTint="?attr/material_theme_secondary_color"
                 app:showAsAction="never" />
             <item
-                android:id="@+id/menu_view_edit_history"
-                android:icon="@drawable/ic_icon_revision_history_apps"
-                app:iconTint="?attr/secondary_text_color"
-                android:title="@string/menu_page_edit_history"
+                android:id="@+id/menu_watch"
+                android:icon="@drawable/ic_star_24"
+                android:title="@string/menu_page_watch"
+                android:visible="false"
+                app:iconTint="?attr/material_theme_secondary_color"
                 app:showAsAction="never" />
             <item
                 android:id="@+id/menu_talk_topic_share"
-                android:icon="@drawable/ic_share_material_secondary_themed"
+                android:icon="@drawable/ic_share"
                 android:title="@string/menu_page_article_share"
+                app:iconTint="?attr/material_theme_secondary_color"
```

Comment:
Looks like we can apply the tint here:
https://github.com/wikimedia/apps-android-wikipedia/pull/3374/files#diff-1e051f2b0330608e92eba8edc616403ec7bdaa76e29ce0cfb794700686715925R118-R121

Do you think it would be better if you update the tint color in your PR? e.g. add `color_group_9` to `android:tint` in `popupTheme`?  

BTW, the color group should be `color_group_9`, per Robin's comment here: 
https://phabricator.wikimedia.org/T303547#7989356

Reply:
Excellent; I've added that style change to this PR.